product/scanning.py

```diff
@@ -21,6 +22,19 @@
 from .model import ContainerImage, Component, UploadResult, UploadStatus
 
 
+class ProcessingMode(Enum):
```

Comment:
I think this should rather be IntEnum (or maybe IntFlag)?
see: https://docs.python.org/3/library/enum.html